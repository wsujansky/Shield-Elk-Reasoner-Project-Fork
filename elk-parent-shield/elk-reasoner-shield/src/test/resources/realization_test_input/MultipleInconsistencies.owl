Prefix(:=<http://example.org/> )
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)

Ontology(

SubClassOf(:A ObjectSomeValuesFrom(:R :B))
SubClassOf(:B ObjectSomeValuesFrom(:R :C))
SubClassOf(:C ObjectSomeValuesFrom(:R :D))
SubClassOf(:D ObjectSomeValuesFrom(:R :A))
SubClassOf(:D owl:Nothing)
ClassAssertion(:A :a)
ClassAssertion(:A :b)
ClassAssertion(:A :c)
ClassAssertion(:A :d)

)
