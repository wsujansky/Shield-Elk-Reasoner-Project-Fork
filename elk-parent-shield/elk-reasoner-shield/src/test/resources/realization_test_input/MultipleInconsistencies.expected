Prefix(:=<http://example.org/> )
Prefix(owl:=<http://www.w3.org/2002/07/owl#>)

Ontology(

EquivalentClasses(owl:Nothing owl:Thing :A :B :C :D)
SameIndividual(:a :b :c :d)
ClassAssertion(:A :a)

)